include/database/DBManager.php

```diff
@@ -1406,14 +1406,14 @@ public function generateInsertSQL(
 						}else{
 							if(isset($type) && $type=='int') {
 								if(!empty($custom_fields[$fieldDef['name']]))
-									$cstm_values[$fieldDef['name']] = $GLOBALS['db']->quote(from_html($val));
+									$cstm_values[$fieldDef['name']] = DBManagerFactory::getInstance()->quote(from_html($val));
```

Comment:
This method should be used a connected DBManager child instance because the DBManager has only an abstract connect() method, only the DBManagerFactory guaranties you it's connected before calling an sql query.

Reply:
OK